docs/userGuide/usingComponents.md

```diff
@@ -205,4 +205,47 @@ If you want to include a keyword without rendering it to the page, add `d-none`
 This is good for catching <span class="keyword">regressions</span>
 ```
 
+# Using Tags
+
+You can use tags to selectively display or hide certain elements on the compiled static site.
+
+Tags are specified by the `tags` attribute, **and can be attached to any HTML element**. For example:
+
+```html
+# Welcome to my site
+
+<p tags="language--english">Hello</p>
+<div tags="language--french">Bonjour</div>
+<span tags="language--spanish">Hola</span>
+```
+
+You need to specify the tags to include in `site.json`, under the `tags` option:
+
+```
+{
+  ...
+  "tags": ["language--english"]
+}
+```
+
+When compiled, only elements that match tags specified in the `site.json` files will be displayed. All other tagged elements will be hidden. In this case, only the element with the `language--english` tag will be displayed. This is helpful when creating multiple versions of a page without having to maintain separate copies. If this option is not specified, all tagged elements will be displayed.
+
+**You can also use multiple tags in a single HTML element. Specify each tag in the `tag` attribute** separated by a space. An element will be displayed if **any of the tags** matches the one in site.json.

```

Comment:
Sorry, just spotted a typo. It should be `tags`, not `tag`? Same as with the code snippet below.

Reply:
To add, can we avoid the term `compiled` as it can be misinterpreted? Also, is it misleading to say show/hide as it sounds like a client-side CSS thing? I assume this is more like include/omit thing that happens at the rendering stage, right?

Reply:
Yup.

Sure, I changed it to `You can use tags to selectively filter certain elements on web pages.`